RR.App/Controllers/HRIS/EmployeeBankingController.cs

```diff
@@ -58,7 +58,7 @@ public async Task<IActionResult> Update([FromBody] EmployeeBankingDto updateEntr
     [HttpGet("details")]
     public async Task<IActionResult> GetBankingDetails([FromQuery] int id)
     {
-        if (_identity.Role is not ("SuperAdmin" or "Admin") && _identity.EmployeeId != id)
+        if (!_identity.IsAdmin && _identity.EmployeeId != id)
```

Comment:
Fix: is not super admin is missing

Reply:
IsAdmin covers both SuperAdmin and admin 
![image](https://github.com/user-attachments/assets/dcd3a096-fa0b-4b77-914d-b911ddfd9e94)